abci/tests/test_cli/test.sh

```diff
@@ -21,8 +21,8 @@ function testExample() {
 	abci-cli --log_level=error --verbose batch < "$INPUT" > "${INPUT}.out.new"
 	killall "$3"
 
-	pre=$(shasum < "${INPUT}.out")
-	post=$(shasum < "${INPUT}.out.new")
+	pre=$(sha256sum < "${INPUT}.out")
```

Comment:
This won't work on macOS (Darwin's analogue is `shasum -a 256`, but I don't know if the Linux versions handle it)

Reply:
what garbage.

Reply:
I will revert this.